mailalerts.php

```diff
@@ -289,15 +289,16 @@ public function hookActionValidateOrder($params)
 					$customization_text = preg_replace('/---<br \/>$/', '', $customization_text);
 			}
 
+        		$link = new Link();
```

Comment:
Hi Eolia,

Thank you for your contribution, why do not use $context->link ?

Best regards.